src/deprecation.hpp

```diff
@@ -15,6 +15,7 @@
 #pragma once
 
 #include <string>
+#include <cstdint>
```

Comment:
Please keep the includes in alphabetical order (unless that requires reordering a lot of already-existing lines).

Reply:
Sure, fixed.